apps/datapuller/src/class.ts

```diff
@@ -0,0 +1,52 @@
+import { IClassItem } from "@repo/common";
+import { ClassesAPI } from "@repo/sis-api/classes";
+
+import setup from "./shared";
+import mapClassToNewClass, { CombinedClass } from "./shared/classParser";
+import { Config } from "./shared/config";
+import { fetchActiveTerms, fetchPaginatedData } from "./shared/utils";
+
+async function updateClasses(config: Config) {
+  const log = config.log;
+  const classesAPI = new ClassesAPI();
+
+  log.info("Fetching Active Terms");
+  const activeTerms = await fetchActiveTerms(log, {
```

Comment:
For updating classes and sections, I would recommend fetching both `Current` and `Next` terms, as future terms are likely to change quite often as well.

Reply:
I believe this is already done in `utils.ts`